src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
+
+    // method to mark task as done
```

Comment:
I like the use of comments for these public methods, perhaps you can take it a step further and make them javadocs comments instead.